src/Components/Nav.tsx

```diff
@@ -27,24 +27,12 @@ export default function Nav(props: Props) {
 					props.setPage('myClowed');
 				}}
 			/>
-			<FontAwesomeIcon
-				icon={faTableList}
-				onClick={() => {
-					props.setPage('leaderboard');
-				}}
-			/>
-			<FontAwesomeIcon
-				icon={faComments}
-				onClick={() => {
-					props.setPage('chat');
-				}}
-			/>
 			<FontAwesomeIcon
 				icon={faGear}
 				onClick={() => {
```

Comment:
Due to the fact that these pages are rendered on top, when I clicked on the button again I expected the overlay to close. This may just be me, but I expected it to work lol